interview-questions.md

```diff
@@ -7,53 +7,53 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is a block in Ruby?
 
-  Your answer:
+  Your answer: Blocks in Ruby are anonymous functions with a do/end statement or occasionally {}.
 
-  Researched answer:
+  Researched answer: Blocks are anonymous, don't strictly belong to an object, and can accept any data-type as input. Since they're anonymous, they can return a value, but lack a name.
```

Comment:
Blocks are a way of modifying a Ruby method. It allows you to create customizable outcomes. This is useful when it comes to iteration.